draft-ietf-httpbis-client-cert-field.md

```diff
@@ -121,35 +121,39 @@ Finished messages.
 
 ## Encoding
 
-The field-values of the HTTP header field defined herein utilize the following
-encoded form.
+The header in this document encodes certificates as Structured Field Byte
+Sequences ({{Section 3.3.5 of RFC8941}}).
 
-A certificate is represented in text as an `EncodedCertificate`, which is the
-base64-encoded (Section 4 of {{!RFC4648}}) DER {{!ITU.X690.1994}} PKIX certificate. The
+The binary sequence is always a DER {{!ITU.X690.1994}} PKIX certificate. The
 encoded value MUST NOT include any line breaks, whitespace, or other additional
-characters. ABNF {{?RFC5234}} syntax for `EncodedCertificate` is shown in the
-figure below.
+characters. A binary sequence which cannot be successfully parsed as a
+certificate MUST be ignored.
 
-~~~
- EncodedCertificate = 1*( DIGIT / ALPHA / "+" / "/" ) 0*2"="
-
- DIGIT = <Defined in Section B.1 of [RFC5234]>  ; A-Z / a-z
- ALPHA = <Defined in Section B.1 of [RFC5234]>  ; 0-9
-~~~
+Note that DER certificates are often stored in a format which is already
+compatible with {{!RFC8941}}; if so, it will be sufficient to replace
+`---(BEGIN|END) CERTIFICATE---` with `:` and remove line breaks in order
+to generate an appropriate item.
```

Comment:
Best I understand things, to most folks "DER certificate" means the binary encoding while "PEM certificate" means the textual encoding you've alluded to here, which is the base64 encoding of the DER binary encoding surrounded with the ---(BEGIN|END) CERTIFICATE--- lines. 

So, while not technically wrong, I worry that it might confuse or mislead. 

Maybe just remove the "DER" from this text?  

Or try and qualify/explain/reference with a bit more specificity? Something like this (using https://datatracker.ietf.org/doc/html/rfc7468#section-5.1 that I just learned existed while searching on the topic):

`Note that certificates are often stored in a format ({{Section 5.1 of RFC7468}}), which is already compatible with {{!RFC8941}};`

Reply:
I'll try and work something up for this over in PR #1874